### PR TITLE
Add December release to home page, fix typo

### PR DESCRIPTION
**Pull request by loarabia**
Added a page describing what was completed in December with pointers to the latest release notes for December to appear as the top item on the homepage.

Fixed a typo in the November post at the same time.

### CODE REVIEW

File: _posts/2019-12-13-december-release.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+title: Azure SDK Releases (December 2019)
+layout: post
+date: 13 Dec 2019
+sidebar: releases_sidebar
+repository: azure/azure-sdk
+---
+
+The Azure SDK team is pleased to announce the December 2019 release of the Azure client libraries.  We have some libraries that have reached General Availability and others that we have provided a new preview.
+
+The following libraries have reached our general availability level.  This requires support from all four languages (.NET, Java, Python, and TypeScript).
```
Comment (jongio) [inline _posts/2019-12-13-december-release.md]: Please use JavaScript/TypeScript or just JavaScript

File: _posts/2019-12-13-december-release.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+* Cosmos
+* Event Hubs
+* KeyVault Certificates
+* Storage DataLake Files
+
+We recommend that you use the generally available client libraries for all new projects, and consider upgrading existing projects to use the new libraries as time permits.
+
+We also encourage you to try out the new previews and give us your feedback by opening an issue on the appropriate GitHub repository.
+
+You can find all the details in our release notes for each language:
```
Comment (jongio) [inline _posts/2019-12-13-december-release.md]: Add: You can find a complete list of all our SDK packages here: https://aka.ms/sdkrel

File: _posts/2019-12-13-december-release.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+* [.NET release notes]({{site.baseurl}}/releases/2019-12/dotnet.html)
+* [Java release notes]({{site.baseurl}}/releases/2019-12/java.html)
+* [Python release notes]({{site.baseurl}}/releases/2019-12/python.html)
+* [TypeScript release notes]({{site.baseurl}}/releases/2019-12/js.html)
```
Comment (jongio) [inline _posts/2019-12-13-december-release.md]: JavaScript/TypeScript or JavaScript

File: _posts/2019-12-13-december-release.md (hunk 2 above)
Comment (jongio) [inline _posts/2019-12-13-december-release.md]: Add: Note about filing issues/suggestions directly on GitHub?

File: _posts/2019-12-13-december-release.md (hunk 1 above)
Comment (loarabia) [reply _posts/2019-12-13-december-release.md]: Will convert to just JavaScript

File: _posts/2019-12-13-december-release.md (hunk 3 above)
Comment (loarabia) [reply _posts/2019-12-13-december-release.md]: Will convert to just JavaScript

File: _posts/2019-12-13-december-release.md (hunk 2 above)
Comment (loarabia) [reply _posts/2019-12-13-december-release.md]: Some language was already there how about this?

"We are always looking for developer feedback. When you have feedback, open an issue on the GitHub repository for the language of the library you're using."

Comment (loarabia) [reply _posts/2019-12-13-december-release.md]: Added

**Pull request merged**